File: gherkin/java/src/main/java/gherkin/ast/Scenario.java
package gherkin.ast;

import java.util.Collections;
import java.util.List;

public class Scenario extends StepsContainer {
    private final List<Tag> tags;
    private final List<ScenarioData> examples;

    public Scenario(List<Tag> tags, Location location, String keyword, String name, String description, List<Step> steps, List<ScenarioData> examples) {
        super(location, keyword, name, description, steps);
        this.tags = Collections.unmodifiableList(tags);
        this.examples = Collections.unmodifiableList(examples);
    }

    public List<Tag> getTags() {
        return tags;
    }

    public List<ScenarioData> getExamples() {
        return examples;
    }
}

File: gherkin/dotnet/Gherkin/Ast/Scenario.cs
using System.Collections.Generic;

namespace Gherkin.Ast
{
    public class Scenario : StepsContainer, IHasTags
    {
        public IEnumerable<Tag> Tags { get; private set; }
        public IEnumerable<Examples> Examples { get; private set; }

        public Scenario(Tag[] tags, Location location, string keyword, string name, string description, Step[] steps, Examples[] examples) 
            : base(location, keyword, name, description, steps)
        {
            Tags = tags;
            Examples = examples;
        }
    }
}